### PR TITLE
Sip 15 tests

### PR DESCRIPTION
**Pull request by hav-noms**

### CODE REVIEW

File: contracts/Liquidations.sol [hunk 1]
```diff
@@ -46,16 +46,22 @@ contract Liquidations is Owned, MixinResolver, ILiquidations {
...
+    uint public constant MIN_LIQUIDATION_DELAY = 86400;   // 1 day
+
     // Storage keys
     bytes32 public constant LIQUIDATION_DEADLINE = "LiquidationDeadline";
     bytes32 public constant LIQUIDATION_CALLER = "LiquidationCaller";
 
+    /* ========== STATE VARIABLES ========== */
     uint public liquidationDelay = 2 weeks; // liquidation time delay after address flagged
-    uint public liquidationRatio = 1e18 / 2; // collateral ratio when account can be flagged for liquidation
+    uint public liquidationRatio = 1e18 * 2; // 200% collateral ratio when account can be flagged for liquidation
```
Comment (jacko125) [inline contracts/Liquidations.sol:64]: ratio's are inverted like collateralRatio / issuanceRatio , so 200% = 0.5 ratio.

Comment (hav-noms) [reply contracts/Liquidations.sol]: cool, reversed this and added liquidationCollateralRatio for the inverse so you can view the liquidationCollateralRatio is 200% while the liquidationRatio is 0.5

File: contracts/Liquidations.sol [hunk 2]
```diff
@@ -124,27 +139,28 @@ contract Liquidations is Owned, MixinResolver, ILiquidations {
...
         liquidationDelay = time;
 
-        // emit event
         emit LiquidationDelayUpdated(time);
     }
 
-    // Collateral ratio is higher when less collateral backing debt
-    // Upper bound is 1.0 (100%)
+    // Accounts Collateral/Issuance ratio is higher when there is less collateral backing their debt
+    // Upper bound issuanceRatio is 1 + penalty (100% + 10% = 110%)
```
Comment (jacko125) [inline contracts/Liquidations.sol:151]: in comment should be upper bound liquidation ratio

Comment (hav-noms) [reply contracts/Liquidations.sol:151]: oh yep

File: contracts/Liquidations.sol [hunk 3]
```diff
@@ -124,27 +139,28 @@ contract Liquidations is Owned, MixinResolver, ILiquidations {
...
     function setLiquidationRatio(uint _liquidationRatio) external onlyOwner {
-        require(_liquidationRatio < MAX_LIQUIDATION_RATIO, "ratio >= MAX_LIQUIDATION_RATIO");
+        require(_liquidationRatio < MAX_LIQUIDATION_RATIO.add(liquidationPenalty), "liquidationRatio >= MAX_LIQUIDATION_RATIO + liquidationPenalty");
```
Comment (jacko125) [inline contracts/Liquidations.sol:153]: Annoying thing about this check is that:

1) If liquidation penalty is updated in separate SCCP, we need to take into factor that it should have a `combined Liquidation Ratio` = 100% + penalty , ie at least 110% to cover collateral and the penalty.  

2) the max value liquidation ratio (equiv of issuance ratio) can be set to,  should be `MAX_LIQUIDATION_RATIO.divideDecimal(SafeDecimalMath.unit().add(liquidationPenalty))` , ie for example it'll be `1 / 1.1 = 0.9090909091` for 110%

File: contracts/Liquidations.sol [hunk 4]
```diff
@@ -212,6 +229,11 @@ contract Liquidations is Owned, MixinResolver, ILiquidations {
         emit AccountRemovedFromLiqudation(_account, now);
     }
 
+    // Returns the inverse view of the issuanceRatio to collateralRatio (1/v)*100
+    function inverseRatio(uint value) internal pure returns (uint) {
+        return (SafeDecimalMath.unit().divideDecimal(value)).multiplyDecimal(1e20);
```
Comment (jacko125) [inline contracts/Liquidations.sol:235]: So this will return the % view of the inverse ratio , ie 200%

File: test/contracts/Liquidations.js [hunk 5]
```diff
@@ -306,22 +417,82 @@ contract('Liquidations', accounts => {
...
+			await addressResolver.importAddresses(['Exchanger'].map(toBytes32), [exchanger.address], {
+				from: owner,
 			});
-			// Alice issues sUSD wen SNX 6 dolla
-			await synthetix.transfer(alice, toUnit('10000'), { from: owner });
+			// now have synthetix resync its cache
+			await synthetix.setResolverAndSyncCache(addressResolver.address, { from: owner });
+		});
+		it('when SNX rate is stale then revert', async () => {
+			fastForward(hour * 4); // 3 hour stale period
```
Comment (jacko125) [inline test/contracts/Liquidations.js:432]: don't forget `await` here

File: test/contracts/Liquidations.js [hunk 6]
```diff
@@ -306,22 +417,82 @@ contract('Liquidations', accounts => {
...
+			await assert.revert(
+				synthetix.liquidateDelinquentAccount(alice, sUSD100, { from: bob }),
+				'Rate stale or not a synth'
+			);
+		});
+		it('when a liquidator has SettlementOwing from hasWaitingPeriodOrSettlementOwing then revert', async () => {
```
Comment (jacko125) [inline test/contracts/Liquidations.js:438]: good checks mocking exchanger

File: contracts/Liquidations.sol (hunk 4 above)
Comment (hav-noms) [reply contracts/Liquidations.sol:235]: i just fixed it to take out the *100

File: test/contracts/Liquidations.js (hunk 5 above)
Comment (hav-noms) [reply test/contracts/Liquidations.js:432]: done

File: test/contracts/Liquidations.js (hunk 6 above)
Comment (hav-noms) [reply test/contracts/Liquidations.js:438]: yeah its nice

**Pull request merged**